File: Lv1/핸드폰_번호_가리기/Solution.js
function solution(phone_number) {

    let answer = '';

    let length = phone_number.length;

    //length - 4 까지 "*"로 치환한다
    let hiddenPart = "*".repeat(length -4);

    let visiblePart = phone_number.slice(-4); // 마지막 4자리 추출
    // -n은 문자열의 끝에서 n번째 문자부터 시작한다는 의미입니다.

    answer = hiddenPart + visiblePart;


    return answer;
}

File: Lv1/핸드폰_번호_가리기/Solution.java
package Lv1.핸드폰_번호_가리기;

class Solution {
    public String solution(String phone_number) {
        String answer = "";
        int length = phone_number.length();
        String hiddenPart = "*".repeat(length-4); //'length - 4'가지 "*"를 반복
        String visiblePart = phone_number.substring(length-4);
        answer = hiddenPart + visiblePart;
        return answer;
    }
}